### PR TITLE
MODSOURMAN-258 - Added jobExecutionProgress schema

### PR DESCRIPTION
**Pull request by RuslanLavrov**

### CODE REVIEW

File: schemas/mod-source-record-manager/jobExecutionProgress.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "$ref": "../common/uuid.json"
+    },
+    "jobExecutionId": {
+      "description": "Corresponding jobExecution id",
+      "type": "string"
+    },
+    "currentSucceeded": {
```
Comment (KaterynaSenchenko) [inline schemas/mod-source-record-manager/jobExecutionProgress.json]: currentlySucceeded
countSucceeded
succeededCounter
??

File: schemas/mod-source-record-manager/jobExecutionProgress.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      "description": "Current number of successfully processed records",
+      "type": "integer",
+      "default": 0
+    },
+    "currentFailed": {
```
Comment (KaterynaSenchenko) [inline schemas/mod-source-record-manager/jobExecutionProgress.json]: currentlyFailed
countFailed
??

File: schemas/dto/rawRecordsMetadata.json [hunk 3]
```diff
@@ -4,6 +4,10 @@
   "type": "object",
   "additionalProperties": false,
   "properties": {
+    "first": {
```
Comment (KaterynaSenchenko) [inline schemas/dto/rawRecordsMetadata.json]: what if first chunk comes not first?

Comment (OleksiiKuzminov) [reply schemas/dto/rawRecordsMetadata.json]: plz add default value

Comment (RuslanLavrov) [reply schemas/dto/rawRecordsMetadata.json]: reverted these changes

**Pull request merged**